src/views/virtualmachines/details/tabs/configuration/network/utils/utils.ts

```diff
@@ -16,3 +23,27 @@ export const isPendingHotPlugNIC = (
   vmi: V1VirtualMachineInstance,
   nicName: string,
 ): boolean => isRunning(vm) && (!isActiveOnGuest(vmi, nicName) || isAbsent(vm, nicName));
+
+export const interfaceNotFound = (vm: V1VirtualMachine, nicName: string) =>
+  !Boolean(getInterfaces(vm)?.find((iface) => iface?.name === nicName));
+
+export const isPendingRemoval = (
+  vm: V1VirtualMachine,
+  vmi: V1VirtualMachineInstance,
+  nicName: string,
+): boolean => {
+  if (!vmi || isStopped(vm)) return false;
+
+  const isVMRunning = isRunning(vm);
+
+  const autoAttach = getAutoAttachPodInterface(vm) !== false;
```

Comment:
Would you mind changing this to `autoAttachPodInterface` just to make it a little clearer?